addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -1262,7 +1262,11 @@ const ColorpickerUserValueWidget = SelectUserValueWidget.extend({
      */
     open: async function () {
         const _super = this._super.bind(this);
-        await this._renderColorPalette();
+        if (this.colorPalette.setSelectedColor) {
+            this.colorPalette.setSelectedColor(this._value);
+        } else {
+            await this._renderColorPalette();
+        }
         _super(...arguments);
     },
```

Comment:
Render in start, then only use setSelectedColor. If there is a real improvement not rendering in start, it can't be done this way: open is not an async function in 14.0, it cannot become one. At worst, it's ok if we only improve that in master and not in 14.0.

Reply:
If we render the 75 palettes in the `start`, it becomes 3.5 times slower. (~300ms -> ~1000ms)
The `setSelectedColor` is only used on `open` which is an almost instantaneous interactive operation.

Reply:
Replaced the `async` function signature by a `setTimeout`.